603-spring-web-smallrye-openapi/src/main/resources/application.properties

```diff
@@ -2,7 +2,6 @@ spring.application.name = Bootstrap Spring Boot
 
 quarkus.datasource.db-kind=h2
 quarkus.datasource.jdbc.url=jdbc:h2:mem:bootapp;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
-%prod.quarkus.datasource.jdbc.url=jdbc:h2:tcp://localhost/mem:test
```

Comment:
Ok, so this URL is heavily outdated anyway (since ~ Quarkus 2.2).

This means that the `quarkus.datasource.jdbc.url` works for both JVM and native, as long as the `@QuarkusTestResource(H2DatabaseTestResource.class)` is present. Is that correct?

And do you know what is the motivation for the `<quarkus.profile>native</quarkus.profile>` in the `native` maven profile?

Reply:
Yes, ` quarkus.datasource.jdbc.url` now works for both JVM and native.

It looks like the intention was to avoid a conflict over ports: https://github.com/quarkus-qe/beefy-scenarios/pull/122